pkg/rancher-desktop/window/index.ts

```diff
@@ -222,18 +222,16 @@ const updateView = (window: any, payload: any) => {
   }
 
   const contentSize = window.getContentSize();
-  const titleBarHeight = 0;
-
   const yZoomFactor = window.webContents.getZoomFactor();
-
   const x = Math.round(payload.x * window.webContents.getZoomFactor());
```

Comment:
Shouldn't this use `yZoomFactor` too? It's the same value for both x and y, right?

Reply:
`yZoomFactor` doesn't really need to be a thing.. we've removed the code that deals with calculating the `yZoomFactor` after we updated electron in 0673f7cac36d05d60237a70e8310fa56936c206d.